xml2kinto/synchronize.py

```diff
@@ -36,4 +36,10 @@ def push_changes(diff, kinto_client, bucket, collection):
         for record in to_create:
             batch.create_record(record)
 
+    if len(to_create) or len(to_delete):
```

Comment:
I don't think you need to check the `len`, if they're empty, they'll still be seen as falsy